src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -125,36 +121,22 @@ public void ValidateConfig()
             IFileSystem fileSystem,
             ILogger logger)
         {
-            // Database Type cannot be null or empty
```

Comment:
The method summary doesn't fall in place with what this method is doing anymore.

Reply:
The validations that this function is doing is related to data-source only. The caller of this method here:
https://github.com/Azure/data-api-builder/blob/f36a660f9182ad17898df9148dd1208d17d50d39/src/Service/Configurations/RuntimeConfigValidator.cs#L98

does only one extra validation - that related to connection string. I think we can club these 2 into one function. The name `ValidateDataSourceInConfig` makes sense.

Reply:
Yeah, if we want to refrain anymore refactors, we should be good to take it outside of this PR.
I saw that there was also an empty string/null check for databasetype removed from this method (maybe it is not required anymore or is placed somewhere else?). That was the only check related to databasetype. Others are more about data source, and hence the comment.